src/main/java/hudson/plugins/jira/JiraSite.java

```diff
@@ -505,6 +505,14 @@ public String getName() {
         return url.toExternalForm();
     }
 
+    /**
+     * @deprecated should not be used
```

Comment:
The comment is quite vague here, so it should not be used but it is used by BlueOcean. 
Why? what needs to happen so it is not used there, should we open some PR to it?